apps/ex_nvr_web/lib/ex_nvr_web/controllers/page_html/webrtc.html.heex

```diff
@@ -0,0 +1,5 @@
+<div>
+  <video id="webRtcPlayer" data-device={@device.id} controls></video>
+  <pre :if={Mix.env() == :dev} id="webRtcLogs" />
```

Comment:
`Mix.env` is not available on releases. Use `Application.get_env(:ex_nvr, :env)` instead.